.github/workflows/ci.yml

```diff
@@ -788,6 +789,7 @@ jobs:
         with:
           path: ".build/bin"
           key: "bin-${{ env.cache-name }}-${{ github.job }}-${{ hashFiles('setup.py') }}\
+-${{ hashFiles('setup.cfg') }}\
```

Comment:
Actually here we can remove both setup.py and setup.cfg. The ./buld/bin content only depends on the version of tools used (helm//kind/kubectl) and it does not depend on setup.py nor setup.cfg in any way. That was probably a copy&paste problem.

Reply:
Will do. (I didn't check the logic here, just saw setup.py being fingerprinted and added setup.cfg)